### PR TITLE
Add Sports Standings app

### PR DESCRIPTION
**Pull request by rs7q5**
This app displays sports standings for various sports (currently only MLB and NHL, sothe important ones :) ). 

currently set to pull every 24 hours as well.

sports_standings

### CODE REVIEW

**Comment by rs7q5** [comment]
If there's enough need, I could add more sports, but I wasn't sure what you're guys' plans were. 

I don't think it would be hard or take long for me to add additional sports, since the frames are created expecting a particular format. 

Also, I didn't want to sit on this while the NHL season is in full swing.

File: apps/sportsstandings/sports_standings.star [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+}
+
+def main(config):
+    sport = config.get("sport") or "MLB"
+    sport_txt, sport_ext = ESPN_SPORTS_LIST.get(sport)
+
+    font = "CG-pixel-3x5-mono"  #set font
+
+    #check for cached data
+    stats_cached = cache.get("stats_rate")
```
Comment (rohansingh) [inline apps/sportsstandings/sports_standings.star:36]: Cache key should include `sport` so we have a different cache for each sport.

```suggestion
    stats_cached = cache.get("stats_rate/%s" % sport)
```

File: apps/sportsstandings/sports_standings.star [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        #get the data
+        if sport == "MLB":
+            stats = get_mlbstats()
+        elif sport == "NHL":
+            stats = get_nhlstats()
+        elif sport == "NBA":
+            stats = get_nbastats()
+
+        #cache the data
+        cache.set("stats_rate", json.encode(stats), ttl_seconds = 86400)  #grabs it once a day
```
Comment (rohansingh) [inline apps/sportsstandings/sports_standings.star:53]:
```suggestion
        cache.set("stats_rate/%s" % sport, json.encode(stats), ttl_seconds = 86400)  #grabs it once a day
```

File: apps/sportsstandings/sportsstandings.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+var source []byte
+
+// New creates a new instance of the Sports Standings applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "sports-standings",
+		Name:        "Sports Standings",
+		Author:      "rs7q5 (RIS)",
+		Summary:     "Get sports standings",
+		Desc:        "Get various sports standings (currently only MLB and NHL).",
```
Comment (rohansingh) [inline apps/sportsstandings/sportsstandings.go:20]: Just updating the description to match the other update.
```suggestion
		Desc:        "Get various sports standings (data courtesy of ESPN).",
```

**Review by rohansingh — CHANGES_REQUESTED** [review]
Just a few minor changes. If they look good to you, please commit them and we can merge. Thanks 😺

**Pull request merged**